### PR TITLE
feat(post): Canepa Open Doors

### PR DESCRIPTION
**Pull request by amalitsky**

### CODE REVIEW

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  '911',
+  '959'
+]
+coverImage: ./service-floor/shop-floor-overview-sq.jpg
+event:
+  date: 2022-08-22T10:00:00-07:00
+  locationName: Canepa
+  locationAddress: 4900 Scotts Valley Drive, Scotts Valley, CA 95066
+description: >-
+  Lets walk through Canepa showroom, museum and shop floor - all full of exotic and classic cars.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:19]: wark through the Canepa

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      - path: ./service-floor/bugatti-eb110-rear.jpg
+        title: Bugatti EB110
+      - path: ./service-floor/porsche-918-spyder-front-sq.jpg
+        title: Porsche 918
+---
+
+## History Bit
+
+[Canepa](https://canepa.com) (also known as _Canepa Design_ and _Canepa Motorsport_)
+is the well-known restoration, tuning and service facility specializing in exotic, classic and
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:111]: is a well-known (introducing for the first time)

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+sports cars.
+Historically their name is strongly connected with
+[Porsche 959](https://en.wikipedia.org/wiki/Porsche_959)
+&mdash; they service, restore, tune and make 959s legal to drive in US since 1988.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:115]: they have been servicing (providing service (not sure)), restoring, tuning etc

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+To this day they have restored almost forty 959s out of approximately 350 ever produced.
+They were involved in passing the [show-n-display](https://en.wikipedia.org/wiki/Show_or_Display)
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:118]: On top of that, they lobbied the passing

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+law in US, which introduced the way to import and register previously non-eligible (not certified)
+but historically significant cars.
+
+Canepa has grown a lot since 1985 and extended their expertise in
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:122]: extended their expertise in various brands - sounds a bit weird. extended their expertise to refine (tune, restore) various brands

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+various European brands &mdash; Ferrari, Porsche, BMW, Mercedes and Bugatti to name a few.
+
+<ImageGrid
+  imageGroups = { props.pageContext.frontmatter.galleryImages[0] }
+></ImageGrid>
+
+Company's office and shop (block of office buildings which were occupied by Seagate)
```
[the remark this answers is not in the document]
Comment (ElenaShavykina) [reply content/posts/2022/canepa-open-house/canepa-open-house.mdx:129]: the block of

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+is located in Scotts Valley - small town on the way from San Francisco South bay to Santa Cruz
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:130]: a small town

Comment (ElenaShavykina) [reply content/posts/2022/canepa-open-house/canepa-open-house.mdx:130]: the San Francisco South bay (see wiki https://en.wikipedia.org/wiki/Santa_Clara_Valley)

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+on the Pacific Coast.
+It is somewhat remote and absolutely beautiful location in Santa Cruz mountains &mdash;
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:132]: the Santa Cruz Mountains (see wiki)

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+far enough from silicon valley buzz, but close enough for the local tow truck hitch-ride.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:133]: (not sure) but the silicon valley buzz

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+
+Spirited drive from Canepa to the nearest ocean beach will take just about twenty minutes.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:135]: One

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 11]
```diff
@@ -0,0 +1,217 @@
...
+Then you can take Pacific Coast Highway up North &mdash; towards Half Moon Bay and San Francisco
+(more traffic) or down South &mdash; to Monterey, Big Sur and San Luis Obispo.
+Proper routes for testing suspension setup and engine tuning.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:138]: testing your(or my) suspension setup and engine tuning

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 12]
```diff
@@ -0,0 +1,217 @@
...
+
+I've been planning to visit them for a while and, once an Open House with Gordon Murray,
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:140]: I had been planning

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 13]
```diff
@@ -0,0 +1,217 @@
...
+visiting on a Monday after the Monterey Car Week was announced, I immediately knew &mdash;
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:141]: visiting on Monday might be omitted - too much information

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 14]
```diff
@@ -0,0 +1,217 @@
...
+the time has come!
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:142]: the time had come (to have all the tenses alligned)

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 15]
```diff
@@ -0,0 +1,217 @@
...
+
+## Canepa Show Room
+
+<ImageGrid
+  imageGroups = { props.pageContext.frontmatter.galleryImages[1] }
+></ImageGrid>
+
+The show room on the first floor is a spacious, if a bit dark,
+area, hosting the rotating exhibit of remarkable cars available for purchase.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:151]: a rotating exhibit - though, you might be right (not sure)

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 16]
```diff
@@ -0,0 +1,217 @@
...
+Every car has a _recommendation letter_ &mdash; spec sheet extended with car's
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:152]: the car's

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 17]
```diff
@@ -0,0 +1,217 @@
...
+After absorbing more recent examples of technological miracles, timeless design
+and engineering beauty, it's time to head upstairs for the refreshing sight of
+various motorsport equipment.
+
+Cars and motorcycles from different eras, competition classes and driven
+by various drivers &mdash; all handpicked by [Bruce Canepa](https://canepa.com/bruce_canepa/)
+&mdash; founder of the company.
+In some of these cars he had competed himself &mdash; like this
+[1980 Pike's Peak Special](https://canepa.com/museum/1980-canepa-pikes-peak-special/) which
+got him the second place overall in 1981.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:172]: which catapulted him ...

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 18]
```diff
@@ -0,0 +1,217 @@
...
+
+<ImageGrid
+  imageGroups = { props.pageContext.frontmatter.galleryImages[3] }
+></ImageGrid>
+
+You can explore the museum collection [online](https://canepa.com/canepa-difference/canepa-museum/).
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:178]: yourselves

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 19]
```diff
@@ -0,0 +1,217 @@
...
+
+## Canepa Shop Floor
+
+Once you are done with the sports cars and bike, go around the staircase
+and through unremarkable door leading to the cat-walk observing the service bay.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:183]: article for the door

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 20]
```diff
@@ -0,0 +1,217 @@
...
+
+It's a lovely sight &mdash; bright, spacious, clean, well organized and full of exotic cars
+of a different age. Show-case of a garage which came straight from heaven.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:186]: article for show-case

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 21]
```diff
@@ -0,0 +1,217 @@
...
+
+From there you can spot a few service bays surrounded with storage shelves used for
+parts taken off cars during extensive restorations and repairs.
+Upholstery area, engine machine shop and paint bays can be seen on the edge of the service zone.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:190]: The

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 22]
```diff
@@ -0,0 +1,217 @@
...
+
+<ImageGrid
+  imageGroups = { props.pageContext.frontmatter.galleryImages[4] }
+></ImageGrid>
+
+All of the above can be explored at any time during regular working hours and
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:196]: the regular working hours

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 23]
```diff
@@ -0,0 +1,217 @@
...
+You can't see it from the observation deck, but further into the shop there is a car storage
+area with the signature 4-level car parking rack.
+
+<ImageGrid
+  imageGroups = { props.pageContext.frontmatter.galleryImages[5] }
+></ImageGrid>
+
+## Wrapping Up
+
+Hope you enjoyed these pictures and I can't recommend the place enough.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:211]: my pictures,

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 24]
```diff
@@ -0,0 +1,217 @@
...
+For one, I will be back to check on new show room arrivals and walk around the shop once again.
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:212]: I will certainly be back

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx [hunk 25]
```diff
@@ -0,0 +1,217 @@
...
+There are so many interesting details there! I certainly got overwhelmed on my first visit.
+
+**Thanks, Canepa, for such an outstanding, accessible and accommodating operation!**
+
+Folks working there are living a dream!
```
Comment (ElenaShavykina) [inline content/posts/2022/canepa-open-house/canepa-open-house.mdx:217]: your dream, you mean))))

**Review by ElenaShavykina — COMMENTED** [review]
please see my comment

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx (hunk 17 above)
Comment (amalitsky) [reply content/posts/2022/canepa-open-house/canepa-open-house.mdx:172]: dogapulted

File: content/posts/2022/canepa-open-house/canepa-open-house.mdx (hunk 21 above)
Comment (amalitsky) [reply content/posts/2022/canepa-open-house/canepa-open-house.mdx:190]: f-ing articles

**Pull request merged**